gax/src/main/java/com/google/api/gax/rpc/ServerStreamingCallable.java

```diff
@@ -42,8 +43,49 @@
  */
 @BetaApi("The surface for streaming is not stable yet and may change in the future.")
 public abstract class ServerStreamingCallable<RequestT, ResponseT> {
+  private final FirstElementCallable<RequestT, ResponseT> firstCallable;
+  private final SpoolingCallable<RequestT, ResponseT> spoolingCallable;
 
-  protected ServerStreamingCallable() {}
+  protected ServerStreamingCallable() {
+    firstCallable = new FirstElementCallable<>(this);
+    spoolingCallable = new SpoolingCallable<>(this);
+  }
+
+  /**
+   * Construct a {@link UnaryCallable} that will yield the first item in the stream and cancel it.
+   * If the stream was empty, the item will be null.
```

Comment:
s/If the stream was empty/If the stream is empty/

Reply:
done